src/Grpc.AspNetCore.Server/README.md

```diff
@@ -4,39 +4,31 @@
 
 ## Configure gRPC
 
-In *Startup.cs*:
+In *Program.cs*:
 
 * gRPC is enabled with the `AddGrpc` method.
-* Each gRPC service is added to the routing pipeline through the `MapGrpcService` method. For information about how to create gRPC services, see [Create gRPC services and methods](https://docs.microsoft.com/aspnet/core/grpc/services).
+* Each gRPC service is added to the routing pipeline through the `MapGrpcService` method. For information about how to create gRPC services, see [Create gRPC services and methods](https://learn.microsoft.com/aspnet/core/grpc/services).
 
 ```csharp
-public class Startup
-{
-    public void ConfigureServices(IServiceCollection services)
-    {
-        services.AddGrpc();
-    }
-
-    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
-    {
-        if (env.IsDevelopment())
-        {
-            app.UseDeveloperExceptionPage();
-        }
-
-        app.UseRouting();
-
-        app.UseEndpoints(endpoints =>
-        {
-            endpoints.MapGrpcService<GreeterService>();
-        });
-    }
-}
+using GrpcGreeter.Services;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddGrpc();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+app.MapGrpcService<GreeterService>();
+app.MapGet("/", () => "Communication with gRPC endpoints must be made through a gRPC client. To learn how to create a client, visit: https://go.microsoft.com/fwlink/?linkid=2086909");
```

Comment:
```suggestion
```
This line isn't needed. Can make the sample look nicer without it.

Reply:
It's in the template for people who try to browse to a gRPC service.

The examples in the readme are to show the minimal amount required in a user-friendly way.